Improve exceptions (#300)
kbsali/php-redmine-api: docs/usage.md

$client->stopImpersonateUser();
```

## Error handling

> Every thrown exception will implement the interface `Redmine\Exception`. This way you can catch all Redmine specific exceptions.

Reviewer: ```suggestion
Every exception implement the interface `Redmine\Exception` making it easy to catch Redmine specific issues.
```
